source/CoordinateConversion/ProAppCoordConversionModule/ViewModels/TabBaseViewModel.cs

```diff
@@ -43,10 +44,14 @@ public TabBaseViewModel()
 
             ListDictionary = new List<Dictionary<string, Tuple<object, bool>>>();
             FieldCollection = new List<object>();
-            Mediator.Register(CoordinateConversionLibrary.Constants.NEW_MAP_POINT, OnNewMapPointInternal);
-            Mediator.RegisterSingleInstance(CoordinateConversionLibrary.Constants.VALIDATE_MAP_POINT, OnValidateMapPointInternal);
-            Mediator.Register(CoordinateConversionLibrary.Constants.MOUSE_MOVE_POINT, OnMouseMoveInternal);
-            Mediator.Register(CoordinateConversionLibrary.Constants.SELECT_MAP_POINT, OnSelectMapPointInternal);
+
+            //Anupam
+            NewMapPointInternal = new RelayCommand(OnNewMapPointInternal);
+            ValidateMapPointInternal = new RelayCommand(OnValidateMapPointInternal);
```

Comment:
@csmoore will cause validation to be run on each derived tab, that would be 2 in it's current state.  Mediator is a generic version of CompositePresentationEvent https://pro.arcgis.com/en/pro-app/sdk/api-reference/#topic7802.html 

Events avoids the need for view models to know about each other and don't break the pattern of MVVM like relay commands use here does.